metrics/prometheus.go

```diff
@@ -1581,6 +1582,40 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 					return values
 				},
 			},
+			{
+				name:        "container_perf_uncore_metric",
```

Comment:
@iwankgb I don't know how I missed this on the perf metrics above, but prometheus metrics should not end in "metric".  Please change the other perf metrics as well.  They should probably be "container_perf_events_total" and "container_perf_uncore_events_total"